doc/src/guides/solving/solve-matrix-equation.md

```diff
@@ -0,0 +1,341 @@
+# Solve a Matrix Equation Algebraically
+
+Use SymPy to solve a matrix (linear) equation. For example, solving $
+\left[\begin{array}{cc} c & d\\1 & -e\end{array}\right] \left[\begin{array}{cc}
+x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ yields
+$ \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc}
+\frac{2e}{ce+d}\\\frac{2}{ce+d}\end{array}\right]$.
+
+## Alternatives to Consider
+- If your matrix and constant vector contain only numbers, not symbols, for
+  example $\left[\begin{array}{cc} 1 & 2\\3 & 4\end{array}\right]
+\left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc}
+  2\\0\end{array}\right]$, you can use one of these other free and open-source
+  packages instead of SymPy:
+    - NumPy's {external:func}`numpy.linalg.solve`
+    - SciPy's {external:func}`scipy.linalg.solve`
+    - mpmath's
+      [lu_solve()](https://mpmath.org/doc/current/matrices.html#linear-equations)
+- Solving a matrix equation is equivalent to solving a system of linear
+  equations, so if you prefer you can
+  [](solve-system-of-equations-algebraically.md)
+- If you formulated your problem as a system of linear equations, and want to
+  convert it to matrix form, you can use {func}`~.linear_eq_to_matrix` and then
+  follow the procedures in this guide.
+
+## Solve a Matrix Equation
+
+Here is an example of solving a matrix equation with SymPy's
+{meth}`sympy.matrices.matrices.MatrixBase.solve`. We use the standard matrix
+equation formulation $Ax=b$ where
+- $A$ is the matrix representing the coefficients in the linear equations
+- $x$ is the column vector of unknowns to be solved for
+- $b$ is the column vector of constants, where each row is the value of an
+  equation
+
+```py
+>>> from sympy import init_printing
+>>> init_printing(use_unicode=True)
+```
+
+```py
+>>> from sympy import symbols
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c,d], [1, -e]])
+>>> A
+⎡c  d ⎤
+⎢     ⎥
+⎣1  -e⎦
+>>> b = Matrix([2, 0])
+>>> b
+⎡2⎤
+⎢ ⎥
+⎣0⎦
+>>> A.solve(b)
+⎡  2⋅e  ⎤
+⎢───────⎥
+⎢c⋅e + d⎥
+⎢       ⎥
+⎢   2   ⎥
+⎢───────⎥
+⎣c⋅e + d⎦
+```
+
+## Guidance
+
+### Matrix Must Be Square
+
+The matrix $A$ must be square to represent a system of linear equations with the
+same number of unknowns as equations. If not, SymPy will give the error
+``ShapeError: `self` and `rhs` must have the same number of rows.``
+
+### Solving Several Matrix Equations With the Same Matrix
+
+If you need to repeatedly solve matrix equations with the same matrix $A$ but
+different constant vectors $b$, it is more efficient to use one of the following
+methods: [LU decomposition](https://en.wikipedia.org/wiki/LU_decomposition) via
+{meth}`~sympy.matrices.matrices.MatrixBase.LUdecomposition`
```

Comment:
Why do you not introduce `LUSolve()` here?

Reply:
That is a great point! I wasn't aware of `LUsolve` when I wrote this section. I revised it to use `LUsolve`.